src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -1,7 +1,7 @@
 // nnet3bin/nnet3-egs-augment-image.cc
 
 // Copyright      2017  Johns Hopkins University (author:  Daniel Povey)
-//                2017  Yiwen Shao
```

Comment:
keep Yiwen's name, I think a little of that code is still there, anyway he will likely add more.